### DISCUSSION TITLE
XR not found on @callstack/react-native-visionos

### DISCUSSION THREAD

**Discussion by mithatb** [callstack/react-native-visionos#99] [Q&A] [answered]
Hello,

On the documentation it says import XR like in below, however there is no XR in the @callstack/react-native-visionos. Do i need to update anything?
import {XR} from "@callstack/react-native-visionos"
Thank you,

**Comment by okwasniewski** [comment, marked as answer]
As mentioned here, #87 this API is not yet released.
